src/main/java/com/ericsson/ec/JenkinsManager.java

```diff
@@ -87,8 +87,7 @@ public JenkinsManager(String protocol, String host, int port, String username, S
     }
```

Comment:
Formatting in this file does not follow Eiffel formatting standards :)

Reply:
I updated the eiffel code style formatter in confluence to support chained method calls.

Reply:
The long javadoc comments.

Reply:
We are using different versions of the format file then. The one I'm using is the one that from the link I sent to everyone in the team.

Reply:
I did notice that there are more files with chained methods though...

Reply:
The java convention file in confluence has a 150 character limit on comments.